config.json

```diff
@@ -1534,6 +1534,17 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "resistor-color",
+      "uuid": "d17bee9c-e803-4745-85ea-864f255fb04e",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "array",
+        "strings"
```

Comment:
`strings` usually refers to exercises that require some sort of string *manipulation*. Since this is just equality-checking, I think we can leave it at just `arrays`.

Reply:
updated!